Default systemd-journal facility to LOG_SYSLOG instead of LOG_LOCAL0

Index: syslog-ng/modules/systemd-journal/journal-reader.c
===================================================================
--- syslog-ng.orig/modules/systemd-journal/journal-reader.c
+++ syslog-ng/modules/systemd-journal/journal-reader.c
@@ -44,7 +44,7 @@
 #define MAX_CURSOR_LENGTH 1024
 
 #define DEFAULT_FIELD_SIZE (64 * 1024)
-#define DEFAULT_PRIO (LOG_LOCAL0 | LOG_NOTICE)
+#define DEFAULT_PRIO (LOG_SYSLOG | LOG_NOTICE)
 #define DEFAULT_FETCH_LIMIT 10
 
 
Index: syslog-ng/modules/systemd-journal/tests/test_systemd_journal.c
===================================================================
--- syslog-ng.orig/modules/systemd-journal/tests/test_systemd_journal.c
+++ syslog-ng/modules/systemd-journal/tests/test_systemd_journal.c
@@ -294,7 +294,7 @@ void
 _test_default_level_test(TestCase *self, TestSource *src, LogMessage *msg)
 {
   gint level = GPOINTER_TO_INT(self->user_data);
-  cr_assert_eq(msg->pri, LOG_LOCAL0 | level, "%s", "Bad default prio");
+  cr_assert_eq(msg->pri, LOG_SYSLOG | level, "%s", "Bad default prio");
   test_source_finish_tc(src);
 }
 
